### ISSUE TITLE
[cocos2dx] Unexpected first frame

### ISSUE THREAD

**Issue by rekcah1986** [EsotericSoftware/spine-runtimes#1216]
Spine: 3.6.53 Professional
Runtime: cocos2d-x 3.17
Spine Runtime: 3.6

![gif](https://user-images.githubusercontent.com/3055444/49058896-fe1de180-f240-11e8-8fde-86e98b559d0f.gif)

Code:
```
    skeletonNode = SkeletonAnimation::createWithBinaryFile("spine/xmfs.skel", "spine/xmfs.atlas", 1.0f);
    skeletonNode->setAnimation(0, "fx_xmfs_ptgj_attack", false);
	    
    skeletonNode->setPosition(Vec2(_contentSize.width / 2, 120));
    skeletonNode->setScale(0.3);
    addChild(skeletonNode);

```
[xmfs.zip](https://github.com/EsotericSoftware/spine-runtimes/files/2618355/xmfs.zip)

**Comment by badlogic** [comment]
I can not reproduce this. I've added your assets to our `CoinExample.cpp` but do not see the superfluous frame you show in your GIF.

Please provide me with code that reproduces the issue reliably and I'll re-open the issue.

**Comment by rekcah1986** [comment]
@badlogic 
Try this code and click it more times:
```
	EventListenerTouchOneByOne* listener = EventListenerTouchOneByOne::create();
	listener->onTouchBegan = [this] (Touch* touch, Event* event) -> bool {
		Director::getInstance()->replaceScene(CoinExample::create()->scene());
		return true;
	};
```

**Comment by rekcah1986** [comment]
[output.zip](https://github.com/EsotericSoftware/spine-runtimes/files/2629156/output.zip)
I uploaded the project file if it's helpful, Thanks for helping.

**Comment by badlogic** [comment]
This has been fixed in the 3.6, 3.7-beta, and 3.7-beta-cpp branches.
